common/StringVector.hpp

```diff
@@ -32,8 +32,9 @@ struct StringToken
 };
 
 /**
- * Safe wrapper around an std::vector<std::string>. Gives you an empty string if you would read past
- * the ends of the vector.
+ * Safe wrapper around an std::vector<std::string>. Gives
```

Comment:
Nit: not sure this is intended, the header in that file says we wrap at 100 chars, the old one seems to be within that limit.